test/test_integration_pumactl.rb

```diff
@@ -197,27 +197,80 @@ def test_clustered_stats
     skip_unless :fork
     skip_unless :unix
 
-    puma_version_pattern = "\\d+.\\d+.\\d+(\\.[a-z\\d]+)?"
+    min_threads = 1
+    max_threads = 2
```

Comment:
is this (intentionally?) changing the behavior of the test?

maybe do that in a different standalone PR so the behavior vs. style change is clear

Reply:
Yes, current code had the following:
```ruby
'running' => 2,
'pool_capacity'  => 2,
'max_threads'    => 2,
```

PR code has:
```ruby
'running' => 1,
'pool_capacity'  => 2,
'max_threads'    => 2,
```

So, 'running' no longer has the same value as 'max_threads'.  Changing 'pool_capacity' is a bit messier in CI...

Reply:
gotcha - why the change?

Reply:
> why the change?

So fewer of the hash values have the same value.  Current code has three with the same value of 2. After the PR, only two values are identical.  Really, all hash values need to be unique for the tests to verify that they're all correct.

Off topic, but with some planned test suite updates, we should be able to have a test file something similar to `test_stats`, where all the 'full' stats tests would be located.